server/mock/presenterteKandidaterMock.js

```diff
@@ -0,0 +1,13 @@
+// https://presenterte-kandidater-api.intern.nav.no:9000/ekstern/antallkandidater?virksomhetsnummer=XXX
+module.exports = {
+    mock: (app) => {
+        app.use(
+            '/min-side-arbeidsgiver/presenterte-kandidater-api/ekstern/antallkandidater',
+            (req, res) => {
+                res.send({
+                    antallKandidater: Math.floor(Math.random() * Math.floor(10))
```

Comment:
```suggestion
                    antallKandidater: Math.floor(Math.random() * 10)
```
Har ikke testet, men `Math.floor(10) = 10`.